vars/piperExecuteBin.groovy

```diff
@@ -46,6 +46,9 @@ void call(Map parameters = [:], stepName, metadataFile, List credentialInfo, fai
             String defaultConfigArgs = getCustomDefaultConfigsArg()
             String customConfigArg = getCustomConfigArg(script)
 
+            echo "stepParameters: ${stepParameters}"
+            echo "PIPER_parametersJSON: ${groovy.json.JsonOutput.toJson(stepParameters)}"
+
```

Comment:
Should be reverted before merging?

Reply:
No, i think this is valuable to understand what happens if you check a failed build. I'd like to keep those messages.

Reply:
I am not sure. The logs become increasingly overfilled with repeated information. Here you log the same thing twice, and often piper itself will output parameters again, and then once more in case of error, and then handlePipelineStepErrors() will output it again for good measure. Maybe I am doing it wrong, but it seems to increase the time it takes me to find the interesting spot in the logs.

Reply:
logging is always a tradeoff and I agree that more log is not always better, but the parameters json needs logging badly imo, also for the sake of local reproducibility of the pipeline. Maybe that point is not valid for stepParameters, if you trust the json serialisation (which should be fine)